src/BaselineOfPharoBootstrap/BaselineOfPharoBootstrap.class.st

```diff
@@ -142,17 +148,26 @@ BaselineOfPharoBootstrap >> baseline: spec [
 			'System-CommandLineHandler'.
 			'System-Finalization'.
 			'System-Hashing'.
-			'System-Model'.
 			'System-Platforms'.
 			'System-SessionManager'.
 			'System-Sources'.
 			'System-Support'.
 
 			'UIManager'.
-			'Zinc-Character-Encoding-Core'.
-			'InitializePackagesCommandLineHandler' }.
+			'Zinc-Character-Encoding-Core' }.
 
 		"These packages are added using hermes after bootstrap"
+
+		spec group: 'AdditionalPackages' with: {
+			'InitializePackagesCommandLineHandler'.
+			'System-Model'.
```

Comment:
Guille: Where is System-Model used?
Pablo: Only by Monticello
Guille: Maybe we can move this to the Monticello initialization and have  one less thing in hermes

Reply:
Ok, I will improve that.